Add more languages support.

lakshayrohila opened the pull request:
Does these things:-

1. Adds fonts-unifont in DISTRO_PKGS_SPECS-ubuntu-jammy (main jammypup64 iso) (might not look very good for few languages, but atleast removes few TOFUs).
2. Adds **fonts-noto-cjk** and **fonts-noto-core** to **nls.sfs** for **JammyPup64**.
3. Adds **gtk3-nocsd**.

lakshayrohila committed:
Update DISTRO_PKGS_SPECS-ubuntu-jammy

lakshayrohila committed:
Add fonts-noto-cjk to nls

lakshayrohila committed:
Move fonts-noto-core to nls.

lakshayrohila committed:
Added unifont.ttf to /usr/share/fonts/default/TTF
Helps to print most of characters of different languages, though being small. Removes Tofu's atleast.
Please refer https://forum.puppylinux.com/viewtopic.php?p=64791#p64791 for more info.

dimkr commented on woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy:
```diff
@@ -63,7 +63,7 @@ yes|bison|bison|exe>dev,dev,doc,nls||deps:yes
 yes|bluez|bluez,bluez-obexd|exe,dev,doc,nls||deps:yes
 no|boehm-gc|libgc1,libgc-dev|exe,dev,doc,nls||deps:yes
-no|busybox||exe
+yes|busybox||exe
```
This conflicts with the busybox petbuild

dimkr commented on woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy:
```diff
@@ -678,7 +681,7 @@ no|xsoldier|xsoldier|exe,dev>null,doc,nls
 no|xvidcore|libxvidcore4,libxvidcore-dev|exe,dev,doc,nls
 yes|xz|xz-utils,liblzma5,liblzma-dev|exe,dev,doc,nls||deps:yes
-no|yad||exe
+yes|yad||exe
```
This conflicts with the yad petbuild

lakshayrohila replied:
Sorry, forgot that I had added those. Will remove them now.

dimkr commented:
Binary files should not be added to woof-CE like this - either add the Ubuntu fonts-unifont package, or add a petbuild that builds this package from source. Binary files make a git repo bigger and slower to clone, because changes are not tracked normally.

lakshayrohila committed:
Resolve conflictions for yad and busybox petbuild

lakshayrohila commented:
> Binary files should not be added to woof-CE like this - either add the Ubuntu fonts-unifont package, or add a petbuild that builds this package from source. Binary files make a git repo bigger and slower to clone, because changes are not tracked normally.

The binary file added is just ~11MBs, and is also TTF. Adding it to woof-code directory should make it usable with all pups. Please correct me if I am wrong, will correct it then.

Thanks

dimkr commented:
> The binary file added is just ~11MBs, and is also TTF.

That's a lot! The repo will grow by 11 MB every time you change this file.

lakshayrohila committed:
Removed binary TTF and added fonts-unifont to in DISTRO_PKGS_SPECS-ubuntu-jammy

dimkr commented:
(Plus, you're forcing this 11 MB on **any** Puppy built using woof-CE - adding fonts-unifont to jammy64 will solve your problem in a way that doesn't increase the size of the repo, doesn't collide with upstream distro font packages and doesn't force anything on other Puppy builds)

lakshayrohila committed:
Merge branch 'testing' of https://github.com/lakshayrohila/woof-CE into testing

lakshayrohila commented:
> (Plus, you're forcing this 11 MB on **any** Puppy built using woof-CE - adding fonts-unifont to jammy64 will solve your problem in a way that doesn't increase the size of the repo, doesn't collide with upstream distro font packages and doesn't force anything on other Puppy builds)

Done the requested. Removed binary file and added fonts-unifont to in DISTRO_PKGS_SPECS-ubuntu-jammy.

dimkr commented:
Do you want this PR to be merged as-is or prefer to leave out some unrelated things like the get_libreoffice change?

lakshayrohila committed:
Removed desk_icon_theme_neon.

lakshayrohila committed:
Remove gnome-keyring.

lakshayrohila commented:
I removed desk_icon_theme_neon and gnome-keyring which seemed extremely unnecessary. get_libreoffice, ghostscript, gparted and htop can be kept. What do you think?

dimkr commented:
> What do you think?

Up to you, I don't mind these changes. Nobody is currently working on jammy64, so you can do pretty much whatever you want to it.

lakshayrohila commented:
I think these can be kept. Especially gparted and get_libreoffice as most of the users will like to have those.

dimkr commented:
Squashed to prevent the 11 MB binary file from making it into the git history and merged. 🎉
